data-plane/receiver/src/test/java/dev/knative/eventing/kafka/broker/receiver/ReceiverTracingVerticleTest.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright © 2018 Knative Authors (knative-dev@googlegroups.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.knative.eventing.kafka.broker.receiver;
+
+import dev.knative.eventing.kafka.broker.contract.DataPlaneContract;
+import dev.knative.eventing.kafka.broker.core.metrics.Metrics;
+import dev.knative.eventing.kafka.broker.core.testing.CloudEventSerializerMock;
+import io.cloudevents.CloudEvent;
+import io.cloudevents.core.v1.CloudEventBuilder;
+import io.cloudevents.http.vertx.VertxMessageFactory;
+import io.micrometer.core.instrument.Meter.Id;
+import io.micrometer.core.instrument.cumulative.CumulativeCounter;
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.exporter.logging.LoggingSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.testing.exporter.InMemorySpanExporter;
```

Comment:
Looking at:

```
[INFO] -------------------------------------------------------------
[ERROR] /home/prow/go/src/knative.dev/eventing-kafka-broker/data-plane/receiver/src/test/java/dev/knative/eventing/kafka/broker/receiver/ReceiverTracingVerticleTest.java:[30,45] package io.opentelemetry.sdk.testing.exporter does not exist
[ERROR] /home/prow/go/src/knative.dev/eventing-kafka-broker/data-plane/receiver/src/test/java/dev/knative/eventing/kafka/broker/receiver/ReceiverTracingVerticleTest.java:[67,11] cannot find symbol
  symbol:   class InMemorySpanExporter
```

and than looking here: https://github.com/eclipse-vertx/vertx-tracing/blob/master/vertx-opentelemetry/pom.xml#L58-L61

I think they are just in the `test` scope for vertx... might we want that JAR on our `pom.xml` too?

Reply:
That should be fixed now